src/Homepage.js

```diff
@@ -1,28 +1,21 @@
-import React, {Component} from 'react';
```

Comment:
Thanks for changing this! Just FYI, this is now a "functional component" and not necessarily anything with hooks :)